src/components/common/ProgressBar/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export type Props = {
+  readonly percentage?: string
+  readonly activeColor?: string
+  readonly showLabel?: boolean
+}
+
+const Wrapper = styled.div<Props>`
```

Comment:
You only do this if it's true that you use all the props in the styled-component, but for example `showLabel` is not used

It's more declarative if you use the ones you need. And then, for example, you would be able to define `percentage` as mandatory, so you can do the `percentage || 0` in the reder function, not in the CSS

Reply:
Makes sense, thank you.